eslint.config.mjs

```diff
@@ -0,0 +1,12 @@
+import { defineConfig, globalIgnores } from 'eslint/config';
+import ts from 'eslint-config-cheminfo-typescript/base';
+
+export default defineConfig(
+  globalIgnores(['coverage', 'lib', 'examples']),
+  ts,
+  {
+    rules: {
+      '@typescript-eslint/no-unnecessary-type-parameters': 'off',
```

Comment:
Do we really have a good reason to disabled this rule?

Reply:
Maybe not, I found the root cause was wrong props in tests